user-tool/tool/src/java/org/sakaiproject/user/tool/UsersAction.java

```diff
@@ -806,12 +806,15 @@ public void doSave(RunData data, Context context)
 
 		// commit the change
 		UserEdit edit = (UserEdit) state.getAttribute("user");
+		String valueEmail = (String)state.getAttribute("valueEmail");
+		String oldEmail = (String)state.getAttribute("oldEmail");
 		if (edit != null)
 		{
 			try
 			{
-				String valueEmail = (String)state.getAttribute("valueEmail");
-				if (StringUtils.trimToNull(valueEmail) != null && EmailValidator.getInstance().isValid(edit.getEid()) && !edit.getEid().equalsIgnoreCase(valueEmail)) {
+				//start this validation only when user has changed the email for the account else skip, also skip for admin user
+				if (!SecurityService.isSuperUser() && StringUtils.trimToNull(valueEmail) != null && StringUtils.trimToNull(oldEmail) != null && !(oldEmail.equals(valueEmail))
```

Comment:
I don't think you need this check as oldEmail will never be `null` because of your check in `readUserForm()`. Don't think it needs changing.

The complexity of this validation code should be refactored to make it all much clearer, but again not something for now.